_studio/shared/umc/io/umc_va/src/umc_va_linux.cpp

```diff
@@ -310,10 +310,10 @@ Status LinuxVideoAccelerator::Init(VideoAcceleratorParams* pInfo)
                                   && ((m_Profile & VA_CODEC) != UMC::VA_H265)
                                   && ((m_Profile & VA_CODEC) != UMC::VA_VP8)
                                   && ((m_Profile & VA_CODEC) != UMC::VA_VP9)
-#ifndef ANDROID
```

Comment:
I understand that Android may not support these codecs, but why disable them here? maybe move somewhere to the upper level (MFX)?

Reply:
Android supports all these codecs and they are not disabled here. needAllocatedSurfaces shows possibility of Adaptive Playback feature

Reply:
ok.